### PR TITLE
add /heealth healthcheck route

### PR DESCRIPTION
**Pull request by satblip**
related to issue #1 

- [x] reside at /health
- [x] accept text/plain content-type
- [x] must be authenticated with the ?token= query parameter, and that token must be equal to the value provided in the AUTH_TOKEN environment variable provided in the .env file.
- [x] respond with 200 OK HTTP code to a GET request
- [x] respond with 403 Forbidden if none of the above apply

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -30,6 +30,28 @@ app.use(async (ctx, next) => {
   next();
 });
 
+app.use(async (ctx, next) => {
+  if (ctx.path !== `/health`) return next();
+
+  let valid = false;
+
+  if (ctx.query.token &&
```
Comment (0xadada) [inline index.js]: you could remove this:
```js
ctx.query.token && 
  ctx.query.token === AUTH_TOKEN &&
```

File: index.js [hunk 2]
```diff
@@ -30,6 +30,28 @@ app.use(async (ctx, next) => {
   next();
 });
 
+app.use(async (ctx, next) => {
```
Comment (0xadada) [inline index.js]: The authentication check is already done on the `app.use` block below this block, so if you would move this block down below the `// authentication` block and remove your `AUTH_TOKEN` check, i'll merge it 👍

**Comment by satblip** [comment]
@0xadada here are the changes :)

**Comment by 0xadada** [comment]
Thanks @satblip
👍

**Comment by 0xadada** [comment]
fixed https://github.com/0xadada/vital-cause/issues/1

**Pull request merged**